### DISCUSSION TITLE
code server installation for windows with no internet connection

### DISCUSSION THREAD

**Discussion by AIgiraffe** [coder/code-server#6640] [Q&A]
when the local machine and host machine don't have internet connection，how can i install code server for host machine? Both machine is windows

**Comment by code-asher** [comment]
They both do not have internet connection?  You will need at least one machine with an internet connection.  Could you explain more about your setup?

**Reply by code-asher** [reply]
Also we do not build for Windows yet, so you would need to build it yourself.
